planemo/shed/__init__.py

```diff
@@ -312,6 +314,10 @@ def upload_repository(ctx, realized_repository, **kwds):
             str(repo_id), tar_path, **update_kwds
         )
     except Exception as e:
+        if isinstance(e, bioblend.ConnectionError) and e.status_code == 400 and \
+                e.body == '{"content_alert": "", "err_msg": "No changes to repository."}':
```

Comment:
BioBlend returns this `ConnectionError` exception whose `body` attribute is the stringified version of the JSON returned by Galaxy. I preferred to be strict to not match other real errors, we can relax this if needed.